src/app/legacy/containers/LivePageMediaPlayer/index.tsx

```diff
@@ -0,0 +1,20 @@
+/** @jsx jsx */
```

Comment:
I'll need to double check this. I _think_ there was something to do Opera Mini not playing ball, but this doesn't seem to have been much of a problem lately.

The Next.js app at least should throw a pretty loud error if it comes across a component it can't render under its default runtime.

Reply:
Forgot I possibly addressed this already: https://github.com/bbc/simorgh/pull/10809/files#diff-ce267c5e6295c50027916a3621a5a2e8525964d5526d3c1a560d52c1eca4e347R1

🤦 I'll get a PR up to remove the other references to `jsxruntime` as I don't think they're needed anymore. Thanks for reminding me of this!